.github/ISSUE_TEMPLATE/pre-release.md

```diff
@@ -0,0 +1,66 @@
+---
+name: Pre-Release Rust Toolchain Update
+about: This template can be used to track the update of the Rust toolchain in this repository as well as all downstream product operators leading up to the next Stackable release
+title: "chore: Update Rust toolchain"
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
+    This was created by an issue template: https://github.com/stackabletech/operator-templating/issues/new/choose.
+-->
+
+## Pre-Release Rust Toolchain Update for Stackable Release XX.(X)X
+
+<!--
+    Replace 'TRACKING_ISSUE' with the applicable release tracking issue number.
+-->
+
+Part of <https://github.com/stackabletech/issues/TRACKING_ISSUE>
+
+> [!NOTE]
+> During a Stackable release we need to ensure that every product operator uses
+> the latest Rust toolchain (used by us). To keep the toolchain in sync across
+> all our operators, we update the version centrally in this repository.
+
+### Update Rust Toolchain
+
+```[tasklist]
+### Tasks in this Repository
+- [ ] Update Rust toolchain in the `config/rust.yaml` file.
+- [ ] Generate downstream PRs using the ["Generate Downstream PRs"](https://github.com/stackabletech/operator-templating/actions/workflows/generate_prs.yml) action.
+- [ ] Merge downstream PRs, see below for more details.
+```
+
+### Merge Downstream PRs
+
+Replace the items in the task lists below with the applicable Pull Requests
+
+<!--
+    The following list was generated by:
+
+    yq '.repositories[].name' config/repositories.yaml \
+    | xargs -I {} echo "- [ ] https://github.com/stackabletech/{}/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files"
```

Comment:
I've been adding `sort` to mine just to ensure consistent ordering (eg: the yaml parsing will be in order of entry and not alphabetical).

```suggestion
    yq '.repositories[].name' config/repositories.yaml \
    | sort \
    | xargs -I {} echo "- [ ] https://github.com/stackabletech/{}/pulls?q=sort:updated-desc+is:pr+is:open+Update+templated+files"
```

Reply:
Good point!